### ISSUE TITLE
Use "Plus" icon instead of "Copy" icon

### ISSUE THREAD

**Issue by johardi** [metadatacenter/cedar-project#1168]
The use of a "copy" icon to enter a new value is not intuitive for users. Using a standard "plus" icon might be better.

<img width="810" alt="Screen Shot 2022-06-10 at 9 34 09 AM" src="https://user-images.githubusercontent.com/5062950/173111881-f7d08d9e-f772-498f-b23a-25f24e50d346.png">

**Comment by graybeal** [comment]
Yes, first person to have the chance to change that should Just Do It. Please!

Should consider whether to not​ copy values when creating the new row. It's possible to create a copy by keyboard copy-paste, and again, it's more intuitive than to have the copied data just appear.
________________________________
From: Josef Hardi ***@***.***>
Sent: Friday, June 10, 2022 9:37 AM
To: metadatacenter/cedar-project ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [metadatacenter/cedar-project] Use "Plus" icon instead of "Copy" icon (Issue #1168)


The use of a "copy" icon to enter a new value is not intuitive for users. Using a standard "plus" icon might be better.

[Screen Shot 2022-06-10 at 9 34 09 AM]<https://user-images.githubusercontent.com/5062950/173111881-f7d08d9e-f772-498f-b23a-25f24e50d346.png>
